src/app/utils/io-ts-type-definitions.ts

```diff
@@ -0,0 +1,8 @@
+import * as t from 'io-ts';
+
+export const NullOrUndefinedAsNull = new t.Type<null, unknown, unknown>(
+    'NullOrUndefinedAsNull',
+    (v): v is null => v == null,
+    (v, ctx) => (v == null || v == undefined ? t.success(null) : t.failure(v, ctx)),
```

Comment:
It works because we don't actually care it is `null` or `undefined`, but you are right that strict equality operator should be used when both `null` and `undefined` are possible values. Change is included in the next commit.